jekyll/_docs/yarn.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: classic-docs
+title: "Using Yarn (the npm replacement) on CircleCI"
+short-title: "Yarn Package Manager"
+categories: [how-to]
+description: "How to use the Yarn package manager on CircleCI."
+---
+
+<img src="{{site.baseurl}}/assets/img/logos/yarn-logo.svg" style="display:block;margin:15px auto;width:40%;min-width:320px;" alt="Yarn Logo" />
+
+[Yarn][yarn-site] is an open-source package manager for JavaScript. Yarn is pre-installed on CircleCI and the packages it installs can be cached. This can potentially speed up builds but, more importantly, can reduce errors related to network connectivity.
+
+[yarn-site]: https://yarnpkg.com/
+
+## Setup
+
+```yaml
+dependencies:
+  override:
+    - yarn
+test:
+  override:
+    - yarn test
+```
+
+When CircleCI detects a JavaScript project, certain commands (like `npm install` or `npm test`) might be run. To use Yarn instead of npm, we override both the `dependencies` and `test` sections.
```

Comment:
I'd rather have the description before the code, but maybe that's just my preference. Otherwise, the code just sort of appears without any context.